### PR TITLE
[KR] 13-3 Translation 

### PR DESCRIPTION
**Pull request by shc443**

### CODE REVIEW

**Review by skyil7 — CHANGES_REQUESTED** [review]
리뷰 완료했습니다. 업데이트 부탁드립니다~ 
번역 수고하셨습니다!

**Comment by Atcold** [comment]
@shc443, any update?

**Comment by shc443** [comment]
Hello Alf!

I finished editing, and I was going to upload it by this weekend.

Let me know if that would work for you.

Have a good rest of your day, and talk to you soon.

Best,
Kevin

**Comment by Atcold** [comment]
Great! Let's have @skyil7 review it!

**Comment by skyil7** [comment]
Seems @shc443 doesn't upload the file yet.

I will merge this PR as soon as they are ready.

**Comment by Atcold** [comment]
Oh, okay. Let's wait for @shc443 then 😀

**Review by skyil7 — APPROVED** [review]

File: docs/ko/week13/13-3.md [hunk 1]
```diff
@@ -0,0 +1,788 @@
...
+
+<!--
+
+### Note about DGL Message and Reduce Functions
+
+>In DGL, the *message functions* are expressed as **Edge UDF**s (User Defined Functions). Edge UDFs take in a single argument `edges`. It has three members `src`, `dst`, and `data` for accessing source node features, destination node features, and edge features respectively.
+
+The *reduce functions* are **Node UDF**s. Node UDFs have a single argument `nodes`, which has two members `data` and `mailbox`. `data` contains the node features and `mailbox` contains all incoming message features, stacked along the second dimension (hence the `dim=1` argument).
+
+`update_all(message_func, reduce_func)` sends messages through all edges and updates all nodes.
```
Comment (Atcold) [inline docs/ko/week13/13-3.md:528]: This sentence has not been translated.

**Comment by Atcold** [comment]
Also, there was no addition to `docs/_config.yml`, so this page was never actually displayed on the website.
Finally, here https://github.com/Atcold/pytorch-Deep-Learning/commit/2a1ac7c2e059f3c1e76f485a5775b658d6ff6893 I tried to fix all the issues I found. Please, have a look at https://atcold.github.io/pytorch-Deep-Learning/ko/week13/13-3/ and see if there are issues.

**Comment by shc443** [comment]
Hello Alf!

I am sorry for missing that part. Probably we missed it because we could
not check them on the website as you pointed out.

https://github.com/Atcold/pytorch-Deep-Learning/pull/792

I just edited the untranslated sentence and created a pr for that.

Hope you have a good rest of the day!

Best,
Kevin

**Pull request merged**